backend/inspirehep/records/receivers.py

```diff
@@ -34,3 +35,4 @@ def index_after_commit(sender, changes):
                 InspireRecord(model_instance.json, model=model_instance).index(
                     force_delete=force_delete
                 )
+                disambiguate_authors.delay(str(model_instance.id))
```

Comment:
I think this needs to be behind a feature flag, as it shouldn't run on prod yet.